### PR TITLE
Fix reference to docker-compose.yml

### PR DESCRIPTION
**Pull request by maw**
Addresses https://github.com/owncloud/docs/issues/2775

### CODE REVIEW

File: modules/admin_manual/pages/installation/docker/index.adoc [hunk 1]
```diff
@@ -15,8 +15,8 @@ The configuration:
 
 == Installation on a Local Machine
 
-To use it, first create a new project directory and download `docker-compose.yml` from
-https://github.com/owncloud-docker/server.git[the ownCloud Docker GitHub repository]
+To use it, first create a new project directory.  Then, copy and paste the sample
```
Comment (mmattel) [inline modules/admin_manual/pages/installation/docker/index.adoc:18]:
```suggestion
To use it, first create a new project directory. Then, copy and paste the sample
```
Removing a double blank...

**Comment by mmattel** [comment]
Seems that backporting down to 10.3 is necessary...
@phil-davis can you take a look if the PR and my bp point is correct?

File: modules/admin_manual/pages/installation/docker/index.adoc (hunk 1 above)
Comment (phil-davis) [reply modules/admin_manual/pages/installation/docker/index.adoc]: done

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Backport to 10.5 in PR #2845 

10.4 has a different way that it is done - https://github.com/owncloud/docs/blame/10.4/modules/admin_manual/pages/installation/docker/index.adoc has line 3:
```
:docker-compose-file-url: https://raw.githubusercontent.com/owncloud/docs/master/modules/admin_manual/examples/installation/docker/docker-compose.yml
```

Somehow that was not in `master` - the change was done in #2232 directly in 10.4 branch and was never ported forward.

I guess we just leave this alone! It is not broken, and gradually the old docs branches will become less relevant for backports...

**Pull request merged**